tensorflow_datasets/image/celebahq.py

```diff
@@ -68,20 +69,23 @@ def __init__(self, resolution, **kwargs):
         version=v2,
         **kwargs)
     self.resolution = resolution
-    self.file_name = "data%dx%d.tar" % (resolution, resolution)
+    self.file_name = "data%dx%d" % (resolution, resolution)
+    self.extensions = resource._KNOWN_EXTENSIONS
```

Comment:
Why is it required ? You should not import protected members.

Reply:
Noted. I will replace it with value of `resource._KNOWN_EXTENSIONS`.
I used it for making the code more generic by supporting all types of archives.